Algorithm.CSharp/CustomDataUsingMapFileRegressionAlgorithm.cs

```diff
@@ -46,7 +46,8 @@ public override void Initialize()
             SetStartDate(2013, 06, 27);
             SetEndDate(2013, 07, 02);
 
-            _symbol = AddData<CustomDataUsingMapping>("FOXA").Symbol;
+            var foxa = QuantConnect.Symbol.Create("FOXA", SecurityType.Equity, Market.USA);
+            _symbol = AddData<CustomDataUsingMapping>(foxa).Symbol;
```

Comment:
this seems wrong -- we create a `Symbol` object and then we pass it to a method that accepts a string and then pull the generated symbol.... why define `foxa` at all? Does the string `"FOXA"` not work?